tests/subscription.test.ts

```diff
@@ -50,10 +52,17 @@ describe("subscriptions.test.ts", () => {
     admin,
     organization,
   }) => {
-    await admin.setSubscriptionState({
-      active: false,
-      ids: [organization.did],
+    const now = new Date();
+    const start = advance(now);
+    const end = advance(start);
+    const txHash = "";
+    const setSubscriptionResponse = await admin.setSubscriptionState({
+      did: organization.did,
+      start,
+      end,
```

Comment:
You might have to convert these into `Date` here, if you end up using Temporal

```suggestion
      start: new Date(start.epochMilliseconds),
      end: new Date(end.epochMilliseconds),
```